cyclops-ui/src/components/pages/Modules/Modules.tsx

```diff
@@ -107,103 +131,200 @@ const Modules = () => {
       );
     }
 
-    return filteredData.map((module: any, index) => (
-      <Col key={index} xs={24} sm={12} md={8} lg={8} xl={6}>
-        <a href={"/modules/" + module.name}>
-          <Card
-            title={
-              <div>
-                {module.iconURL ? (
-                  <img
-                    alt=""
-                    style={{ height: "2em", marginRight: "8px" }}
-                    src={module.iconURL}
-                  />
-                ) : (
-                  <></>
-                )}
-                {module.name}
-              </div>
-            }
-            style={{
-              borderLeft: "solid " + getStatusColor(module) + " 5px",
-              width: "100%",
-              maxWidth: "500px",
-            }}
-            className={styles.modulecard}
-          >
-            <Row gutter={[16, 16]}>
-              <Col
-                span={24}
-                style={{
-                  whiteSpace: "nowrap",
-                  overflow: "hidden",
-                  textOverflow: "ellipsis",
-                  display: "block",
-                }}
-              >
-                Repo:
-                <Link aria-level={3}
-                 href={module.template.repo}
-                 target="_blank"
-                 >
-                  {" " + module.template.repo}
-                </Link>
-              </Col>
-            </Row>
-            <Row gutter={[16, 16]}>
-              <Col
-                span={24}
-                style={{
-                  whiteSpace: "nowrap",
-                  overflow: "hidden",
-                  textOverflow: "ellipsis",
-                  display: "block",
-                }}
-              >
-                Path:
-                <Link
-                  aria-level={3}
-                  target="_blank"
-                  href={
-                    module.template.repo +
-                    `/tree/` +
-                    getLinkPath(
+    if (displaymodule.length === 0) {
```

Comment:
Check this comment https://github.com/cyclops-ui/cyclops/pull/437#discussion_r1686814924

Reply:
Have changed the code accordingly